### PR TITLE
[ALLUXIO-2979] Extend the validateEnv command to check the ramdisk is accessible on worker

### PR DESCRIPTION
**Pull request by bf8086**
https://alluxio.atlassian.net/browse/ALLUXIO-2979

This change added validation task to check whether Alluxio worker can mount ram disk on the configured mounting path. The task checks first whether the ram disk is already mounted at the path, and then checks sudo privilege for mounting ramfs if it has not been mounted, or checks write permission if it is mounted.

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/validation/RamDiskMountPrivilegeValidationTask.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+          // If the RAM disk is already mounted, it must be writable for Alluxio worker
+          // to be able to use it.
+          if (!file.canWrite()) {
+            System.err.format("RAM disk at %s is not writable.%n", path);
+            return false;
+          }
+
+          System.out.format("RAM disk is mounted at %s, skip validation.%n", path);
+          return true;
+        }
```
Comment (yupeng9) [inline shell/src/main/java/alluxio/cli/validation/RamDiskMountPrivilegeValidationTask.java]: Should it fail when the mounting point is not ram disk?

Comment (bf8086) [inline shell/src/main/java/alluxio/cli/validation/RamDiskMountPrivilegeValidationTask.java]: If the mounting point is not a ram disk, Alluxio worker can still mount a ramdisk on the directory provided it has the privilege, so we will just fallback to check the sudo privilege below.

File: shell/src/main/java/alluxio/cli/validation/RamDiskMountPrivilegeValidationTask.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      return true;
+    }
+
+    if (path.isEmpty()) {
+      System.err.println("Mount path is empty.");
+      return false;
+    }
+
+    if (path.split(",").length > 1) {
+      System.out.println("Multiple storage path for memory tier found. Skip validation.");
```
Comment (bf8086) [inline shell/src/main/java/alluxio/cli/validation/RamDiskMountPrivilegeValidationTask.java]: Fixed.

**Comment by bf8086** [comment]
@yupeng9 I fixed the typo and rebased on the new classes in server/common, please take a look.

File: shell/src/main/java/alluxio/cli/validation/RamDiskMountPrivilegeValidationTask.java (hunk 1 above)
Comment (yupeng9) [inline shell/src/main/java/alluxio/cli/validation/RamDiskMountPrivilegeValidationTask.java]: that's right.

**Pull request merged**